params/config.go

```diff
@@ -285,7 +284,6 @@ var (
 			UseZktrie:                 true,
 			MaxTxPerBlock:             &ScrollMaxTxPerBlock,
 			MaxTxPayloadBytesPerBlock: &ScrollMaxTxPayloadBytesPerBlock,
-			FeeVaultAddress:           &ScrollFeeVaultAddress,
```

Comment:
```
FeeVaultAddress:           &rcgt.ScrollFeeVaultAddress,
```

otherwise it will make `chainConfig.Scroll.FeeVaultAddress` (which we use extensively) return `nil` 
and `chainConfig.Scroll.FeeVaultEnabled()` return `false`